cmd/query/app/flags_test.go

```diff
@@ -35,9 +35,9 @@ func TestQueryBuilderFlags(t *testing.T) {
 	})
 	qOpts, err := new(QueryOptions).InitFromViper(v, zap.NewNop())
 	require.NoError(t, err)
-	assert.Equal(t, "/dev/null", qOpts.StaticAssets.Path)
-	assert.True(t, qOpts.StaticAssets.LogAccess)
-	assert.Equal(t, "some.json", qOpts.UIConfig)
+	assert.Equal(t, "/dev/null", qOpts.UIConfig.AssetsPath)
+	assert.True(t, qOpts.UIConfig.LogAccess)
+	assert.Equal(t, "some.json", qOpts.UIConfig.ConfigFile)
```

Comment:
UIConfig prefix not needed

Reply:
@yurishkuro its needed here because `UIConfig` is not embedded in `QueryOptions` - its only embedded in `StaticAssetsHandlerOptions`